src/styles.css

```diff
@@ -1,3 +1,7 @@
+html {
+  scroll-behavior: smooth;
+}
+
 /* desktop body styles */
 body {
   background-color: #000000;
```

Comment:
Ignore this if the view is just busted, but there's no closing tag on the body?

Reply:
Are you referring to the body tag in the index.html?

Reply:
Sorry, not closing tag - closing bracket. It doesn't look like you have a close after `body {`

Reply:
It's showing up in my code. I'm not sure why you're unable to see it.